### PR TITLE
Adjust skipped profiles

### PR DESCRIPTION
**Pull request by JianLi-RH**
/cc @jianlinliu @shellyyang1989 @evakhoni @jiajliu @dis016

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/tools/get_missed_upgrade_jobs.py [hunk 1]
```diff
@@ -51,13 +51,13 @@ def get_missed_profiles(e2e_jobs, upgrade_jobs):
             - as: vsphere-upi-zones-f28
     """
     missed_profiles = []
-    skipped_profiles = 'disasterrecovery|longduration|destructive|hypershift|ocm|rosa|aro|ui'
+    skipped_profiles = 'agent|alibaba|aro|destructive|disasterrecovery|hypershift|longduration|ocm|rhel|rosa|ui|winc'
```
Comment (shellyyang1989) [inline ci-operator/config/openshift/openshift-tests-private/tools/get_missed_upgrade_jobs.py:54]: I'm wondering why rhel, ui and winc?

Comment (JianLi-RH) [reply ci-operator/config/openshift/openshift-tests-private/tools/get_missed_upgrade_jobs.py:54]: For winc: https://redhat-internal.slack.com/archives/CF8SMALS1/p1713842442500839?thread_ts=1713841950.285019&cid=CF8SMALS1

Comment (JianLi-RH) [reply ci-operator/config/openshift/openshift-tests-private/tools/get_missed_upgrade_jobs.py:54]: rhel: I think `rhel workers are being deprecated in 4.16` https://redhat-internal.slack.com/archives/CF8SMALS1/p1715306294842649, and this script is most likely be used in future work, so we can skip rhel in it

Comment (JianLi-RH) [reply ci-operator/config/openshift/openshift-tests-private/tools/get_missed_upgrade_jobs.py:54]: ui: I think the jobs which name include `ui` means they are ui team owned jobs, we don't need to add them in upgrade. need ui team to confirm. @yapei

Comment (yapei) [reply ci-operator/config/openshift/openshift-tests-private/tools/get_missed_upgrade_jobs.py:54]: May I know what are we going to achieve by this PR?

Comment (JianLi-RH) [reply ci-operator/config/openshift/openshift-tests-private/tools/get_missed_upgrade_jobs.py:54]: @yapei This script is used to find the jobs/profiles which include in e2e config file but not in upgrade config file. In other word, we want to find out the profiles which have not implemented in upgrade.

Comment (yapei) [reply ci-operator/config/openshift/openshift-tests-private/tools/get_missed_upgrade_jobs.py:54]: thanks for clarification @JianLi-RH 
okay to skip jobs with `ui` in it

**Comment by JianLi-RH** [comment]
If there are no objections, please help review and merge this pr

**Comment by shellyyang1989** [comment]
/lgtm

@liangxia PTAL. Thank you!

File: ci-operator/config/openshift/openshift-tests-private/tools/get_missed_upgrade_jobs.py (hunk 1 above)
Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/tools/get_missed_upgrade_jobs.py:54]: Maybe also need to include cloud/mco/netobserv/hive/wrs/security-profiles-operator/fips-check/obo/day2-64k-pagesize/nokubeadmin

Comment (JianLi-RH) [reply ci-operator/config/openshift/openshift-tests-private/tools/get_missed_upgrade_jobs.py:54]: I have confirmed with @lihongyan1 for `obo`, it should be automated for upgrade job.

**Pull request merged**